pkgs/applications/office/paperwork/paperwork-gtk.nix

```diff
@@ -27,12 +28,19 @@ let
     imagemagick
     perlPackages.Po4a
   ];
+  inherit (import ./src.nix { inherit fetchFromGitLab; }) version src sample_documents;
 in
 
 python3Packages.buildPythonApplication rec {
-  inherit (import ./src.nix { inherit fetchFromGitLab; }) version src sample_documents;
+  inherit src version;
   pname = "paperwork";
 
+  sample_docs = sample_documents // {
+    # a trick for the update script
+    name = "sample_documents";
+    src = sample_documents;
```

Comment:
Why exactly is this needed?

Reply:
because `update-source-version paperwork.sample_docs` evaluates the src.url, src.outputHash, name and version attributes

Reply:
Do you consider that a blocker for merging?

Reply:
Nope, although I'd probably would have used a more detailed comment.